go/chat/boxer_test.go

```diff
@@ -700,8 +700,9 @@ func TestChatMessageDeletedNotSuperseded(t *testing.T) {
 		// Delete the body
 		boxed.BodyCiphertext.E = []byte{}
 
-		_, err = boxer.unbox(context.TODO(), *boxed, key)
-		require.Error(t, err, "should not unbox with deleted but no supersededby")
+		unboxed, err := boxer.unbox(context.TODO(), *boxed, key)
+		require.NoError(t, err, "suprisingly, should be able to unbox with deleted but no supersededby")
```

Comment:
There's some context in this PR for why this is true? Maybe summarize it here in a comment?